air/src/options.rs

```diff
@@ -275,6 +294,7 @@ impl Serializable for ProofOptions {
         target.write_u8(self.fri_remainder_max_degree);
         target.write_u8(self.partition_options.num_partitions);
         target.write_u8(self.partition_options.hash_rate);
+        target.write_u8(self.batching_deep as u8);
```

Comment:
We should be able to write this as `target.write(self.batching_deep)`.

Reply:
forgot to change this after adding (de)-serialization to the relevant struct.
Changed now